### PR TITLE
Add some stub documentation for how to use maintainers.

### PR DESCRIPTION
**Pull request by stephenjust**
# Why are we doing this?
It's not always easy for new developers to understand how to use maintainers. Let's start to document it!

# Whats changing?

# Questions/notes for reviewers

# How this was tested
- [ ] unit tests added
- [ ] tested on robot

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206398294118271

### CODE REVIEW

**Comment by stephenjust** [comment]
image

**Comment by stephenjust** [comment]
image

File: src/main/java/xbot/common/command/package-info.java [hunk 1]
```diff
@@ -1,4 +1,66 @@
...
+ * <p>
+ * There are many components of the robot that need to maintain a particular state.
+ * For example, an arm might need to maintain a particular angle, or a drive train
+ * might need to maintain a particular velocity. We can implement these components
+ * as <em>setpoint subsystems</em> that have a <em>maintainer</em> that runs in the
+ * background and keeps the subsystem and its outputs at the desired state.
+ * </p>
+ * <p>
+ * Setpoint subsystems are implemented as {@link xbot.common.command.BaseSetpointSubsystem}s.
+ * They have a {@link xbot.common.command.BaseMaintainerCommand} that runs on the subsystem.
```
Comment (aschokking) [inline src/main/java/xbot/common/command/package-info.java:14]: ⭐ mention that this is subclassed, we don't actually run BaseMaintainerCommands

**Review by aschokking — COMMENTED** [review]
Thanks for adding this! Definitely been a huge gap

**Review by aschokking — APPROVED** [review]

**Review by JohnGilb — APPROVED** [review]

File: src/main/java/xbot/common/command/package-info.java [hunk 2]
```diff
@@ -1,4 +1,66 @@
 /**
  * The command framework, scheduler and subsystems.
+ *
+ * <h2>Setpoint Subsystems and Maintainers</h2>
```
[the remark this answers is not in the document]
Comment (stephenjust) [reply src/main/java/xbot/common/command/package-info.java:4]: My hope is that if the documentation is here, there's a chance Copilot will reada and understand it.

**Pull request merged**